api/views/assets.py

```diff
@@ -444,3 +445,55 @@ def get(self, request):
         ] = 'attachment; filename="sample_import_file.csv"'
 
         return response
+
+
+class ExportAssetsDetails(APIView):
+    permission_classes = [IsAuthenticated, IsAdminUser]
+    authentication_classes = (FirebaseTokenAuthentication,)
+
+    def get(self, request):
+        assets = models.Asset.objects.all()
+        serializer = AssetSerializer(assets, many=True)
```

Comment:
does this limit by asset location? we only want to export assets from the logged in users's location.

Reply:
It doesn't limit assets by location. From the story description on pivotal tracker, it states all assets.

Reply:
let's limit by the users location to avoid confusion when the user sees an asset on file that isn't on the app (since on the web app/API we filter by the users location).

Reply:
ok then. working on it.